spec/lib/eventbrite_spec.rb

```diff
@@ -0,0 +1,48 @@
+require "climate_control"
+require "json"
+require "webmock/rspec"
+require "eventbrite"
+
+describe Eventbrite do
+  around do |example|
+    ClimateControl.modify EVENTBRITE_ACCESS_TOKEN: 'bar' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.